explorer/lib/explorer_web/live/pages/home/index.html.heex

```diff
@@ -10,33 +10,29 @@
     >
       <%= if @operators_registered != :empty do %>
         <%= @operators_registered %>
-      <% else %>
-        0
       <% end %>
     </.card_link>
     <.card_link 
-      icon="hero-arrow-right-solid" 
-      navigate={~p"/restake"} 
-      title="Total Restaked" 
-      subtitle={"(#{@restaked_amount_eth |> Helpers.format_number()} ETH)"}>
-      <%= if @restaked_amount_eth != :nil do %>
-        <%= @restaked_amount_usd |> Helpers.format_number() %> USD
-      <% else %>
-        0
-      <% end %>
+        icon="hero-arrow-right-solid" 
+        navigate={~p"/restake"} 
+        title="Total Restaked" 
+        subtitle={
+          if @restaked_amount_eth |> Helpers.format_number() != nil do
+            "(#{@restaked_amount_eth |> Helpers.format_number()} ETH)"
+          end
+        }>
+        <%= if @restaked_amount_usd |> Helpers.format_number() != nil do %>
```

Comment:
Shouldn't we remove the `|> Helpers.format_number()` here?

Reply:
It's there to cover the `:empty` case more easily I think. The format helper will return `nil` if either `:nil` or `:empty`.